cmd/apps/certmanager_app.go

```diff
@@ -151,7 +152,7 @@ const CertManagerInfoMsg = `# Get started with cert-manager here:
 
 # Check cert-manager's logs with:
 
-kubectl logs -n cert-manager deploy/cert-manager -f`
+Kubectl logs -n cert-manager deploy/cert-manager -f`
```

Comment:
Why is this capitalised? This won't work

Reply:
Done